containers/src/Kokkos_Vector.hpp

```diff
@@ -221,6 +221,8 @@ class vector : public DualView<Scalar*, LayoutLeft, Arg1Type> {
   size_type span() const { return DV::span(); }
   bool empty() const { return _size == 0; }
 
+  iterator data() const { return DV::h_view.data(); }
```

Comment:
Make sure you update the [Synopsys](https://github.com/kokkos/kokkos/wiki/Kokkos%3A%3Avector#synopsis) in the API reference accordingly